src/style.css

```diff
@@ -2,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
```

Comment:
Similarly, add `font-display: swap;` to this `@font-face` rule to enhance performance and user experience.